apps/studio/src/server/modules/resource/resource.service.ts

```diff
@@ -595,6 +599,20 @@ export const publishPageResource = async ({
       return
     }
 
+    // Only create a CodeBuild job if the site has a CodeBuild project associated with it
+    const { codeBuildId } = await getSiteNameAndCodeBuildId(siteId)
+    if (codeBuildId && addCodebuildJobRow) {
+      await tx
+        .insertInto("CodeBuildJobs")
+        .values({
+          resourceId,
+          siteId,
+          userId: user.id,
+          isScheduled,
+        })
+        .execute()
+    }
```

Comment:
comment: this api feels very "fat" in the sense that alot of the control flow variable exposed to the caller seem to be what the method should, in and of itself, handle properly. 

eg: for this variable, i thought the conditional should be whether this is a scheduled publish (unless we also want to inform users of non-scheduled publish, but then that raises the question of what happens if i publish a great # of pages at once... but then in that case, then we don't need this variable `addCodebuildJobRow` anymore)

(edit: went to check the call sites) i think we should just shift this in since this is a feature flag and the caller shouldn't care about htis.